pkg/env/managed.go

```diff
@@ -4,6 +4,9 @@ var (
 	// ManagedCentral is set to true to signal that the central is running as a managed instance.
 	ManagedCentral = RegisterBooleanSetting("ROX_MANAGED_CENTRAL", false)
 
+	// SensorUpgraderEnabled controls whether the secured cluster auto-upgrader is enabled
+	SensorUpgraderEnabled = RegisterBooleanSetting("ROX_SENSOR_UPGRADER_ENABLED", true)
```

Comment:
Given the recent discussion in acs-eng-discuss about the reporting of feature flags:
Would we want to reconsider and change this to `SENSOR_UPGRADER_DISABLED` with default `false` to stick to the proposal?

Reply:
Yeah, I saw the proposal, but I do not agree to apply it here - it would disable the func that was partially enabled before. I don't think it fits to this usecase.